src/components/EmptyList/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function EmptyList() {
+  const history = useHistory();
+
+  const goToAddItemView = () => {
+    history.push('/add-item');
+  };
+
+  return (
+    <section>
+      <p>Your shopping list is currently empty.</p>
+      <button onClick={goToAddItemView}>Add Item</button>
```

Comment:
What's the reason for not using an [anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) instead?

Unless you have a strong reason to use a button that behaves like an anchor, you should use an anchor.

Reply:
Well, we used a button because there was a suggestion in our acceptance criteria about a button. But, we changed `<button> `to `<Link>` of React Router, and surprise! . It's more simple and less code 😄 . Thanks @alejandronanez ! great observation 👍